test/test.ps1

```diff
@@ -2,10 +2,10 @@
 #requires -version 5
 param(
     # Set to 'microsoft' on build servers
-    [string]$RootImageName='test',
+    [string]$RootImageName = 'test',
     # Set to Docker host IP if running within a container
-    [string]$HostIP='localhost',
-    $Folder='*'
+    [string]$HostIP = 'localhost',
+    [string]$Folder = '*'
```

Comment:
The missing type was causing the type to be inferred.  Because of this, when run with `1.0` it was also testing the `1.1` images because `1.1/jessie/sdk -like 1*`